### PR TITLE
Fix app insights

### PR DESCRIPTION
**Pull request by timja**
Correct app insights integration

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@ services:
     environment:
       - DEFINITION_STORE_DB_HOST=ccd-definition-store-database
       - DEFINITION_STORE_DB_PORT=5432
-      - DEFINITION_STORE_DB_USERNAME
-      - DEFINITION_STORE_DB_PASSWORD
+      - DEFINITION_STORE_DB_USERNAME=definition_store
```
Comment (tzarsmango) [inline docker-compose.yml]: why the db user/pass change

File: docker-compose.yml [hunk 2]
```diff
@@ -35,8 +36,8 @@ services:
       timeout: 10s
       retries: 10
     environment:
-      - DEFINITION_STORE_DB_USERNAME
-      - DEFINITION_STORE_DB_PASSWORD
+      - DEFINITION_STORE_DB_USERNAME=definition_store
```
Comment (tzarsmango) [inline docker-compose.yml]: why the db user/pass change

File: lib/AI-Agent.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ApplicationInsightsAgent>
+    <AgentLogger>INFO</AgentLogger>
+    <Instrumentation>
+        <BuiltIn enabled="true">
+            <HTTP enabled="true"/>
+        </BuiltIn>
+        <Class name="uk.gov.hmcts.cmc.claimstore.repositories.DBCaseRepository">
```
Comment (tzarsmango) [inline lib/AI-Agent.xml]: please check the class name

Comment (dwaynebailey) [reply lib/AI-Agent.xml]: @mario-paniccia is busy fixing those.

**Pull request closed without merging**